### ISSUE TITLE
TinyEmberPlus doesn't support trees with several root nodes

### ISSUE THREAD

**Issue by Vasilich** [Lawo/ember-plus#57]
i have scanned one device, that has several root nodes, with EmberPlusView, and saved it in *.EmBER file.
After loading this file in TinyEmberPlus it shows only first root node.
It is even impossible to create second root node manually.
I attached scan of one of the devices with 3 root nodes.
[192.168.0.1.zip](https://github.com/Lawo/ember-plus/files/763066/192.168.0.1.zip)

**Comment by larsbiermann** [comment]
Hi,
I experienced the same problem. Are there any plans to solve this issue?

Thanks, Lars

**Comment by MartinP1** [comment]
Is it a valid approach, to define more than one root node for a device?

I guess it is, since EmberPlusViewer shows up an according "multi" tree...

**Comment by larsbiermann** [comment]
> Is it a valid approach, to define more than one root node for a device?
> 
> I guess it is, since EmberPlusViewer shows up an according "multi" tree...

Yes, it is. The Ember+ Spec says that a root node can be a sequence of RootElements.

![grafik](https://github.com/user-attachments/assets/9de9dd26-4b0f-4220-9beb-88feed0c2632)
